bot.py

```diff
@@ -20,10 +26,17 @@
         async for log in client.logs_from(message.channel, limit=100):
             if log.author == message.author:
                 counter += 1
-
         await client.edit_message(tmp, 'You have {} messages.'.format(counter))
     elif message.content.startswith('!sleep'):
         await asyncio.sleep(5)
         await client.send_message(message.channel, 'Done sleeping')
-
+    elif message.content.startswith("!roll"):  # Example: "!roll 3d6".
+	DICEROLL = preparse(message.content[6:])
```

Comment:
You're adding in tabs (`\t`) here, but the code uses 4 spaces (`    `) to comply with [PEP 8](https://www.python.org/dev/peps/pep-0008/).